tests/JsonApiErrorMiddlewareTest.php

```diff
@@ -0,0 +1,216 @@
+<?php
+
+namespace WtfPhp\JsonApiErrors\Tests;
+
+use Exception;
+use PHPUnit\Framework\TestCase;
+use Psr\Http\Message\ResponseFactoryInterface;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+use WtfPhp\JsonApiErrors\JsonApiErrorMiddleware;
+use WtfPhp\JsonApiErrors\JsonApiException;
+use WtfPhp\JsonApiErrors\Tests\Fakes\TestRequest;
+use WtfPhp\JsonApiErrors\Tests\Fakes\TestResponseFactory;
+
+class JsonApiErrorMiddlewareTest extends TestCase
+{
+    protected ServerRequestInterface $request;
+
+    protected ResponseFactoryInterface $responseFactory;
+
+    protected JsonApiErrorMiddleware $middleware;
+
+    protected function setUp(): void
+    {
+        $this->request = new TestRequest();
+        $this->responseFactory = new TestResponseFactory();
+        $this->middleware = new JsonApiErrorMiddleware($this->responseFactory);
+    }
+
+    /** @test */
+    public function itShoudHandleAnExceptionWithoutErrorCodeAndMessage()
```

Comment:
itShouldHandleAnExceptionWithoutErrorCodeAndMessage - typo 😉

Reply:
Could we remove the `should`? `itHandlesAnExceptionWithoutErrorCodeAndMessage`? 🤔